src/software/ai/navigator/obstacle/obstacle_factory.h

```diff
@@ -94,29 +94,44 @@ class ObstacleFactory
      *
      * @return rectangular obstacle
      */
-    ObstaclePtr createObstacleFromRectangle(const Rectangle &rectangle);
+    ObstaclePtr createObstacleFromRectangle(const Rectangle &rectangle) const;
 
    private:
     std::shared_ptr<const ObstacleFactoryConfig> config;
-    double shape_expansion_amount;
+    double obstacle_expansion_amount;
```

Comment:
nit: this seems like a very generic name for what seems to mostly be for representing robots. Could it be renamed to `robot_expansion_amount` or similar?

Reply:
changed to robot_radius_expansion_amount